### ISSUE TITLE
The role of "graph_model.add(dc.nn.BatchNormalization())" in GraphCov model

### ISSUE THREAD

**Issue by chf42** [deepchem/deepchem#767]
Hi all, 

I am trying to implement dc.models.MultitaskGraphClassifier model to classify one-task case for my own data. 

![image](https://user-images.githubusercontent.com/7895104/29379273-ef57f87e-828f-11e7-82db-84109b3ffc1d.png)

I found that when I didn't use "dc.nn.BatchNormalization()" layer, the prediction (cohen kappa score) for my CNN model is almost 0.0 (i.e. random prediction).  After I add "dc.nn.BatchNormalization()" layer, the prediction for CNN model increase to above 0.4. I was quite surprised by such a difference in model performance. Thus, I am quite curious about **what is the exact role of BatchNormalization layer, & it is always necessary to add when implementing CNN model with DeepChem?**

Thanks,
Cheng

**Comment by rbharath** [comment]
Try using the `TensorGraph` graph convolution (as in our IPython notebook on graph convolutions). The `dc.nn.SequentialGraph` code is now deprecated.

I don't believe I've seen the same dependence on batch normalization with the neww code.

**Comment by rbharath** [comment]
Closing this issue since the older code has now been removed from HEAD.
